wiki/Ranking_criteria/en.md

```diff
@@ -42,6 +42,7 @@ Important to understand before using:
 ### Guidelines
 
 - **Creators of the media used in a beatmap should be credited in the beatmap's description.** Per the [Content Usage Guidelines](/wiki/Rules/Content_usage_guidelines), media used in beatmaps should be permitted for use in osu!, and their licensing will likely detail how to credit their creators. Common reasons to omit credits from the description include exceeding difficulty in sourcing, creators' wishes to remain uncredited, and presence of credits in the artist, title, or source fields.
+  - Beatmaps using AI generated backgrounds should instead credit it as such in the description.
```

Comment:
would it make sense to put "media" here instead of "backgrounds"?

Reply:
Back in the meeting we wanted to narrow the scope to backgrounds for now. Using "media" would potentially include music as well, and we discussed that songs/maps/etc... weren't mature enough to consider as of now.

Reply:
I don't get the differentiation. Please use "media" as it should t matter what the content is.

Reply:
It can be done, but this may cause some confusion that was previously discussed. Do AI vocals on, for example, AI covers count? In practice, we would only be making sure the backgrounds and/or videos which is what we currently can work with.

Changing it to "media" would need further discussion, as this was not what was originally proposed. I'll gather some more opinions on it.

Reply:
Resolved with the most recent update.